tree_sitter_v/src/node-types.json

```diff
@@ -3912,11 +3954,11 @@
   },
   {
     "type": "int_literal",
-    "named": true
+    "named": false
   },
   {
     "type": "int_literal",
-    "named": false
+    "named": true
```

Comment:
Why are these changed? These seem unrelated to the PR to me 🤔 .

Reply:
This file is generated by tree-sitter, I think these tow objects simply exchanged their order.